src/pages/Task.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { useEffect } from "react";
+import { RouteComponentProps } from "react-router-dom";
+import { useParams, useHistory } from "react-router-dom";
+
+const DEFAULT_TAB = "logs";
+
+const Task: React.FC<RouteComponentProps> = () => {
+  const params = useParams<{ tab?: string; taskID: string }>();
+  const history = useHistory();
+  useEffect(() => {
+    if (!params.tab) {
+      history.replace(`/task/${params.taskID}/${DEFAULT_TAB}`);
+    }
+  }, [params.tab, params.taskID, history]);
```

Comment:
prefer destructuring the values of `params` instead of using multiple dot notations. 

`const {tab, taskId} = useParams<{ tab?: string; taskID: string }>();`